app/src/main/java/de/xikolo/models/certificates/CertificateUrls.java

```diff
@@ -0,0 +1,17 @@
+package de.xikolo.models.certificates;
+
+import com.squareup.moshi.Json;
+
+import io.realm.RealmObject;
+
+public class CertificateUrls extends RealmObject {
+
+    @Json(name = "confirmation_of_participation")
+    public String confirmationOfParticipation;
+
+    @Json(name = "record_of_achievement")
+    public String recordOfAchievement;
+
+    @Json(name = "qualified_certificate")
+    public String qualifiedCertificate;
+}
```

Comment:
Just place this model inside the enrollment model class, so you can have a `Enrollment.Certificates` identifier as they are specific to it.